### PR TITLE
Polymer 2 upgrade

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: template/mainwindow.html [hunk 1]
```diff
@@ -129,20 +129,24 @@
         PapanLocalize
       ],
 
-      attached: function() {
+      ready: function() {
```
Comment (tania-m) [inline template/mainwindow.html]: Why not connectedCallback (cf connectedCallback )? Why the switch to ready?

File: template/mainwindow.html [hunk 2]
```diff
@@ -129,20 +129,24 @@
...
         if (PapanUtils.isElectron()) {
           this._toggleDebug = () => {
             const electron = require('electron')
             electron.remote.getCurrentWindow().toggleDevTools()
           }
         } else {
-          document.getElementById('github-link').style.display = 'inline'
+          var githublink = this.$['github-link'].style.display = 'inline'
```
Comment (tania-m) [inline template/mainwindow.html]: Small little detail: "let" instead of "var" :).

File: src/games/tic_tac_toe/render/tic_tac_toe.html [hunk 3]
```diff
@@ -129,7 +129,7 @@
         this._publicScene = {}
       },
 
-      attached: function() {
+      ready: function() {
```
Comment (tania-m) [inline src/games/tic_tac_toe/render/tic_tac_toe.html]: Introduce the usage of ES6 classes for Polymer component definition? Or not? (Polymer2 supports the usage of classes).

File: template/mainwindow.html (hunk 2 above)
Comment (nicolasnoble) [inline template/mainwindow.html]: Done

File: template/mainwindow.html (hunk 1 above)
Comment (nicolasnoble) [inline template/mainwindow.html]: That seemed to have been the conversion manual from the website ?

File: src/games/tic_tac_toe/render/tic_tac_toe.html (hunk 3 above)
Comment (nicolasnoble) [inline src/games/tic_tac_toe/render/tic_tac_toe.html]: That'd be for a later time; that's just a simple direct conversion.

**Pull request merged**